libs/geography/src/models/subdivision.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * The subdivisions of a country as per ISO-3166-2
+ * For the United States this is commonly termed as "States"
+ */
+export interface DaffSubdivision {
+  id: string;
+  name: string;
+  name_en: string;
```

Comment:
Do you think that `name_en` field should be nullable? Magento for example does not provide a separate english name field for [Region](https://devdocs.magento.com/guides/v2.3/graphql/queries/directory-countries.html#region). LMK what you think.

Reply:
fixed